api/v1beta1/access_control_validate.go

```diff
@@ -275,14 +294,23 @@ func isRoleNameValid(roleName string) (bool, error) {
 
 // Indicates if privilege is a valid privilege.
 func isPrivilegeValid(
-	privilege string, aerospikeConfigSpec AerospikeConfigSpec,
+	privilege string, aerospikeConfigSpec AerospikeConfigSpec, version string,
 ) (bool, error) {
 	parts := strings.Split(privilege, ".")
 
 	_, ok := Privileges[parts[0]]
 	if !ok {
-		// First part of the privilege is not part of defined privileges.
-		return false, fmt.Errorf("invalid privilege %s", privilege)
+		cmp, err := asconfig.CompareVersions(version, "6.0.0.0")
+		if err != nil {
+			return false, err
+		}
+		if cmp < 0 {
+			// First part of the privilege is not part of defined privileges.
+			return false, fmt.Errorf("invalid privilege %s", privilege)
+		}
+		if _, ok := Post6Privileges[parts[0]]; !ok {
+			return false, fmt.Errorf("invalid privilege %s", privilege)
+		}
 	}
 
```

Comment:
since Privileges is not the complete list of all privileges the code that follows will be incorrect for post60 privilege. See line 324

Reply:
See the previous comment on making Privileges as the complete set.